flytekit/core/workflow.py

```diff
@@ -60,6 +60,9 @@
 from flytekit.models.documentation import Description, Documentation
 from flytekit.types.error import FlyteError
 
+if TYPE_CHECKING:
+    from flytekit.tools.translator import Options
```

Comment:
Why this need `if TYPE_CHECKING`?

Reply:
This is used to prevent circular import if two modules are mutually dependent.

Reply:
this is just a tiny dataclass that depends only on the model files... can we just move it out somewhere please?  then we don't need this type checking.

Reply:
@wild-endeavor I’m planning to move Options into a new file (flytekit/core/options.py). Should we provide a shortcut for users to easily import it using:
```
from flytekit import Options
```
Currently, users have to import it using:
```
from flytekit.core.options import Options
```
or even
```
from flytekit.tools.translator import Options
```
The latter feels odd since I’m relocating Options from that file, but many existing docs still use this import.